libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -101,6 +100,11 @@ namespace eosio { namespace testing {
          map<transaction_id_type, transaction_receipt> chain_transactions;
    };
 
+   class tester : public base_tester {
+   public:
```

Comment:
Should we move push_genesis_block to this class from base_tester?

Reply:
Yes and done.